docs/images.md

```diff
@@ -128,6 +179,9 @@ pulled using imagePullSecrets.  That is, imagePullSecrets does *NOT* protect you
 images from being seen by other users in the cluster.  Our intent
 is to fix that.
 
+You can use this in conjunction with a per-node `.dockerfile`.  The credentials
```

Comment:
Would it be worth putting this in a `**Note for GKE:**` block at the top of this section as well? I'm not sure whether this would lead people to ask 'why don't they put "Note for AWS," "Note for XYZ," etc.', but if I'm thinking purely about document consistency, it would be great to have, under each section, a note about whether I can use this for my provider. You've done this for the negatives, so I'm just wondering whether the positives would also be helpful.